api/framer-motion.api.md

```diff
@@ -260,6 +260,11 @@ export interface FeatureProps extends MotionProps {
 // @public (undocumented)
 export type ForwardRefComponent<T, P> = ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>>;
 
+// Warning: (ae-internal-missing-underscore) The name "FramerTreeContext" should be prefixed with an underscore because the declaration is marked as @internal
+// 
+// @internal (undocumented)
+export const FramerTreeContext: import("react").Context<SyncLayoutBatcher | SharedLayoutSyncMethods>;

```

Comment:
Should we prefix this with an underscore?

Reply:
I don't have a strong opinion here but we have several other exports that are @internal that aren't prefixed with an underscore, so in the interest of keeping things consistent I would prefer not to add an underscore.